node/events.ts

```diff
@@ -82,10 +82,27 @@ export class EventEmitter {
   }
 
   private maxListeners: number | undefined;
-  private _events: EventMap;
+  private _events!: EventMap;
+
+  static #init(emitter: EventEmitter): void {
+    if (
+      emitter._events == null ||
+      emitter._events === Object.getPrototypeOf(emitter)._events // If `emitter` does not own `_events` but the prototype does
+    ) {
+      emitter._events = Object.create(null);
+    }
+  }
+
+  /**
+   * Overrides `call` to mimic the es5 behavior with the es6 class.
+   */
+  // deno-lint-ignore no-explicit-any
```

Comment:
I think this is not ideal. However, I'm not sure if `EventEmitter` should be rewritten in ES5 style... :thinking:

Reply:
I think it's ok with either way as long as it passes the test cases